src/components/albums/albums.service.js

```diff
@@ -0,0 +1,20 @@
+export class AlbumsService {
+  constructor($q, $http) {
+    'ngInject;'
+    this.$q = $q;
+    this.$http = $http;
+    this.baseUrl = '/search?q=';
+  }
+  
+  getAlbums(albumName) {
```

Comment:
On this case, don't make sense to return a promise if $http is already a promise, you can return $http directly